src/Spec/validator-generated.php

```diff
@@ -5270,6 +5274,78 @@ public static function createValidationRules() {
   $o_1287->code = ValidationErrorCode::CSS_SYNTAX_INVALID_ATTR_SELECTOR;
   $o_1287->format = 'CSS syntax error in tag \'%1\' - invalid attribute selector.';
   $o_0->error_formats[] = $o_1287;
+
+      $o_9999 = new TagSpec();
```

Comment:
`valiadator-generated.php` is a generated file. See https://github.com/Lullabot/amp-library/blob/master/src/Spec/README.md.

It's probably due for an update. Although the fork meant to produce this file is several commits behind the latest amp validator https://github.com/Lullabot/amphtml :/. That's beyond this issue obviously.

Maybe helpful, is that I just merged a change to `\Lullabot\AMP\AMP` to allow rules to be injected. Take a look at `\AmpTest::getParsedRulesTestSet` to see how that's used. You could potentially make use of that for customizations like this.